### DISCUSSION TITLE
Deezer !

### DISCUSSION THREAD

**Discussion by labaland** [music-assistant/support#1191] [Feature requests and ideas]
Hi!

Is it please possible to add deezer as provider?

It is evern possible to download 320kbps / FLAC from deezer with 3thirdparty programs. Must be easy to add support then??  i dont know :) 

Maybe theese sites/files can help some?

I pray for it 🙏 

## Deezer is available https://music-assistant.io/music-providers/deezer/

**Comment by OzGav** [comment]
Please don’t spam multiple discussions. Stick with the existing one. And if you had looked a the music provider poll you would see that Deezer is under development

**Comment by jonbng** [comment]
As of version 2.0.0 beta 30 music assistant has support for Deezer as a music provider. More information here: https://github.com/orgs/music-assistant/discussions/1245
